### ISSUE TITLE
Cells with emoji are not formatted properly

### ISSUE THREAD

**Issue by DutchPete** [tgrosinger/advanced-tables-obsidian#33]
**Describe the bug**

*Even after formatting, not everything is formatted.

**To Reproduce**

*Make a table, fill the cells, and do a format.
This bug occurs both with "format at cursor" and "format all tables".*

**Expected behavior**

*All pipes to line up*

**Screenshots**

<img width="425" alt="table1" src="https://user-images.githubusercontent.com/10794265/100546165-6e183580-3268-11eb-8a54-108b24f368ef.png">

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOSX
 - Obsidian Version: [e.g. v0.9.10] (Settings → About → Current Version) 0.9.19
 - Advanced Tables Version: [e.g. 0.4.0] (Settings → Third-party plugin → Browse → Advanced Tables → currently installed) 0.7.0

**Additional context**

**Comment by tgrosinger** [comment]
@DutchPete, thanks for the bug report. I think this is because emoji often consist of two characters which are rendered as a single character. Making these work in monospace fonts might be tricky... but I will give it a shot.

**Comment by DutchPete** [comment]
@tgrosinger
> … but I will give it a shot.

Tony, any chance to have a look at that?

**Comment by tgrosinger** [comment]
Oh, no I have not had a chance to yet. Thanks for the reminder. I will make an effort to look into it this week.

**Comment by tgrosinger** [comment]
@DutchPete, I'm having trouble reproducing the problem you are experiencing. For me the columns are not quite perfectly aligned with the emoji, but it's not like the example you showed me where they are completely misaligned.

<img width="372" alt="Screen Shot 2020-12-06 at 3 10 52 PM" src="https://user-images.githubusercontent.com/597386/101295920-587eaf00-3818-11eb-96aa-36ef2379d29d.png">

Can you please copy and paste your whole example table into this ticket (surround in a code block). Please also let me know if you have changed the font via either a css snippet, or in the Advanced Tables Plugin Setting (see image below).

<img width="809" alt="Screen Shot 2020-12-06 at 3 13 26 PM" src="https://user-images.githubusercontent.com/597386/101295963-aabfd000-3818-11eb-8ecb-2b0e953d2466.png">

**Comment by DutchPete** [comment]
@tgrosinger : your last question (monospace) triggered me to check. When I 1<sup>st</sup> installed your plug-in we discussed the font and you made me understand it would have to be a monospace font, so I looked for one that also looks nice in Preview - remember I try to emulate WYSIWYG.

So I found and installed *Inconsolata*. But somehow that was deleted somewhere along the line without me realizing it. Today I checked, saw the box was empty so monospace was back to default. I put Incolsolata back, and ……… bingo, problem solved !! It is almost perfect ! And that font is a lot nicer than the default font.

Many thanks for your help, and apologies for having disturbed you 😳
